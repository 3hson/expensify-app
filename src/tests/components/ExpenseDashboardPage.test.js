import React from 'react';
import{shallow} from 'enzyme';
import ExpenseDashbordPage from '../../components/ExpenseDashbordPage';
import expenses from '../fixtures/expenses';

test('this must render ExpenseDashboard',()=>{
    const wrapper =shallow(<ExpenseDashbordPage  />);
    expect(wrapper).toMatchSnapshot()
})