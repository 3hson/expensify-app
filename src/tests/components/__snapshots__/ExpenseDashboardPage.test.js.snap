// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this must render ExpenseDashboard 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
